index.js

```diff
@@ -0,0 +1,7 @@
+module.exports = algoliasearch;
+
+function algoliasearch(applicationID, apiKey, opts) {
+  var AlgoliaSearch = require('./src/algoliasearch');
+
+  return new AlgoliaSearch(applicationID, apiKey, opts);
```

Comment:
I usually put the defaults in this kind of function so it separates the code of the constructor from the noise induce by defaults.